.circleci/config.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       # Rebuild fnserver if necessary
       - run: |
           if [[ -n "$FN_NEEDED" ]]; then
-            make test -j $(nproc)
+            sudo PATH=/usr/local/go/bin:$PATH make test -j $(nproc)
```

Comment:
do we need sudo to run tests now? is this line still necessary?

Reply:
running fn server should not cause any troubles, cause we are it in dind container as root, the iofs ownership change will work.

I think this is more about fn dev experience, especially when running tests locally :-(

Reply:
yea, I agree. i'm curious what our options are. it may be interesting to explore what docker themselves do [for local dev/test], they have a similar thing to us in a few ways between the unix socket and the docker user (e.g. docker run requires sudo w/o adding the docker user to your group)

Reply:
without sudo we get:

`msg="failed to change owner" app_id=01D8SBQ92T1A70320ZJ0000002 cpus= error="chown /tmp/iofs310208425: operation not permitted"`

see https://github.com/fnproject/fn/pull/1478

we could also disable the security feature all together, but I would probably disable it by default and enable it in circleci only